testsuite/MDAnalysisTests/converters/test_openmm_parser.py

```diff
@@ -130,6 +160,82 @@ class TestOpenMMTopologyParser(OpenMMTopologyBase):
     expected_n_bonds = 1922
 
 
+class TestOpenMMTopologyParserWithNoElements(OpenMMTopologyBase):
+    ref_filename = app.PDBFile(PDB).topology
+    for a in ref_filename.atoms():
+        a.element = None
+
+    expected_n_atoms = 47681
+    expected_n_residues = 11302
+    expected_n_segments = 1
+    expected_n_bonds = 25533
+
+    expected_attrs = [
+        "ids",
+        "names",
+        "resids",
+        "resnames",
+        "masses",
+        "bonds",
+        "chainIDs",
+    ]
+
+    def test_no_elements_warn(self, top):
+        wmsg = ("Element information is missing, elements attribute "
+                "will not be populated. "
+                "Atomtype attribute will be guessed using atom "
+                "name and mass will be guessed using atomtype."
+                "See MDAnalysis.topology.guessers.")
+
+        with pytest.warns(UserWarning, match=wmsg):
+            mda_top = self.parser(self.ref_filename).parse()
+
+
+class TestOpenMMTopologyParserWithPartialElements(OpenMMTopologyBase):
+    ref_filename = app.PDBFile(PDB).topology
+    expected_n_atoms = 47681
+    expected_n_residues = 11302
+    expected_n_segments = 1
+    expected_n_bonds = 25533
+
+    def test_with_partial_elements(self, top):
+        if 'elements' in self.expected_attrs:
+            wmsg = ("Unknown element None found for some atoms. "
+                    "These have been given an empty element record "
+                    "with their atomtype set to 'X' "
+                    "and their mass set to 0.0. "
+                    "If needed they can be guessed using "
+                    "MDAnalysis.topology.guessers.")
+            with pytest.warns(UserWarning, match=wmsg):
+                mda_top = self.parser(self.ref_filename).parse()
+                assert mda_top.types.values[3344] == 'X'
+                assert mda_top.types.values[3388] == 'X'
+                assert mda_top.elements.values[3344] == ''
+                assert mda_top.elements.values[3388] == ''
+
+    def test_abnormal_elements(self, top):
+        omm_top = Topology()
+        bad1 = Element(0, "*", "*", 0*daltons)
+        bad2 = Element(0, "?", "?", 6*daltons)
+        silver = Element.getBySymbol('Ag')
+        chain = omm_top.addChain()
+        res = omm_top.addResidue('R', chain)
+        omm_top.addAtom(name='Ag', element=silver, residue=res)
+        omm_top.addAtom(name='Bad1', element=bad1, residue=res)
+        omm_top.addAtom(name='Bad2', element=bad2, residue=res)
```

Comment:
Thanks for adding this test of invalid elements! Could you please just add an atom without an element as well, to test that :)